### PR TITLE
SalesWings Actions docs deploy

### PR DESCRIPTION
**Pull request by markzegarelli**
- Add SalesWings destination docs
- Extend docs
- Cleanup
- Cleanup
- Fix grammar
- Add missing hyphens
- Minor cleanup
- Minor cleanup
- Metadata and clean up

### Related issues (optional)
Continues from #3970

### CODE REVIEW

**Comment by markzegarelli** [comment]
@denis-egorushkin-sw I've created this branch so I could push some edits and updates to the doc. I've rewritten a few areas from clarity, and created a table from the list provided. We've also added the destination's Preset and Action settings, which will render when the page is built.

We'll deploy Tuesday, Jan 31st in the morning, Pacific time.

File: src/connections/destinations/catalog/actions-saleswings/index.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+| Segment Event | `Kind` value | `Data` value                                                | Custom Event Activity         |
+| ------------- | ------------ | ----------------------------------------------------------- | ----------------------------- |
+| Track         | Track        | The name of the Track event, for example, `User Registered` | `[Track] User Registered`     |
+| Identify      | Identify     | The email address as identified by Segment                  | `[Identify] peter@example.com |
+| Screen        | Screen       | The name of the screen                                      | `[Screen] Home View`          |
+
+You can override these defaults in the action configuration and map `Kind` and `Data` to static values, or map them to other properties that are part of the Segment event.
+
+All the defaults can be overridden in the action configuration, you can either set `Kind` and `Data` to static values, or map them to string properties from Segment events.
```
Comment (denis-egorushkin-sw) [inline src/connections/destinations/catalog/actions-saleswings/index.md:48]: Hi @markzegarelli, thank you for the changes.

Once question: the second sentence seems to be redundant now, can we remove it?

Comment (markzegarelli) [inline src/connections/destinations/catalog/actions-saleswings/index.md:48]:
```suggestion

```

File: src/connections/destinations/catalog/actions-saleswings/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+{% include content/plan-grid.md name="actions" %}
+
+[SalesWings](https://www.saleswingsapp.com/){:target="_blank"} is a lead scoring platform that offers a user-friendly, no-code solution to identify your leads' true interests. The SalesWings Destination enables using the data collected in Segment to identify, tag and prioritize your leads in SalesWings for your Marketing and Sales teams.
+
+## Getting started
+
+1. From the Segment web app, click **Catalog**, then click **Destinations**.
+2. Find the Destinations Actions item in the left navigation, and click it.
+3. Click **Configure SalesWings (Actions)**.
+4. Select an existing Source to connect to SalesWings (Actions).
```
Comment (stayseesong) [inline src/connections/destinations/catalog/actions-saleswings/index.md:18]:
```suggestion
1. From the Segment web app, navigate to **Connections > Catalog** and select the **Destinations** tab of the catalog. 
2. Search for **SalesWings (Actions)** and select the destination. 
3. Click **Configure SalesWings (Actions)**.
4. Select an existing Source to connect to SalesWings (Actions).
```

**Review by stayseesong — APPROVED** [review]

File: src/connections/destinations/catalog/actions-saleswings/index.md (hunk 1 above)
Comment (markzegarelli) [reply src/connections/destinations/catalog/actions-saleswings/index.md]: Yes, thank you! Removed.

**Pull request merged**